test/utils/fizzy_engine.cpp

```diff
@@ -23,9 +23,10 @@ std::unique_ptr<WasmEngine> create_fizzy_engine()
     return std::make_unique<FizzyEngine>();
 }
 
-void FizzyEngine::parse(bytes_view input)
+bool FizzyEngine::parse(bytes_view input)
 {
     m_instance.module = fizzy::parse(input);
```

Comment:
You should catch exception here and return false then.

Reply:
Added.